pkg/schedule/operator/operator_controller.go

```diff
@@ -580,6 +583,19 @@ func (oc *Controller) removeOperatorLocked(op *Operator) bool {
 	return false
 }
 
+func (oc *Controller) removeRelatedOperatorLocked(op *Operator) {
+	relatedID, _ := strconv.ParseUint(op.AdditionalInfos[string(RelatedRegionID)], 10, 64)
+	if relatedOp := oc.operators[relatedID]; relatedOp != nil && relatedOp.Status() != CANCELED {
+		log.Info("operator canceled related region",
+			zap.Uint64("region-id", op.RegionID()),
+			zap.String("additional-info", relatedOp.GetAdditionalInfo()),
+			zap.Duration("takes", relatedOp.RunningTime()))
+		relatedOp.Cancel(RelatedRegionID)
+		oc.removeOperatorLocked(relatedOp)
```

Comment:
Do we need to reorder these two like others do?

Reply:
I don't get what you mean, can u give me more info...?

Reply:
change the sequence of these two lines